src/classes/FORM_ServiceGiftEntry_TEST.cls

```diff
@@ -206,6 +206,81 @@ public with sharing class FORM_ServiceGiftEntry_TEST {
             'New template names should return true from `FORM_ServiceGiftEntry.checkNameUniqueness`');
     }
 
+    @isTest
+    static void shouldRetrieveTemplateWithCRUDErrors() {
+        UTIL_PermissionsMock utilPermissionsMockInstance = new UTIL_PermissionsMock(false, 'CRUD');
+        UTIL_Permissions utilPermissionsMock = (UTIL_Permissions) Test.createStub(UTIL_Permissions.class,
+            utilPermissionsMockInstance);
+        UTIL_Permissions.setInstance(utilPermissionsMock);
+        
+        FORM_Template template = createSampleTemplate();
+
+        createSampleFieldMappings();
+        
+        FORM_Template updatedTemplate = FORM_ServiceGiftEntry.checkPermissions(template,
+            new Set<FORM_PermissionValidator.AccessLevel> {
+                FORM_PermissionValidator.AccessLevel.VALIDATE_READ,FORM_PermissionValidator.AccessLevel.VALIDATE_CREATE,
+                FORM_PermissionValidator.AccessLevel.VALIDATE_UPDATE});
+
+        System.assert(String.isNotBlank(updatedTemplate.permissionErrors), 'There should be permission errors   when the user has no CRUD access to an object.');
```

Comment:
See above, 'CRUD'.

Reply:
fixed. no longer need 'CRUD' in this class as its being used in `FORM_PermissionsValidator_TEST` for more robust testing there.